db/migrate/20160512012637_add_thumbnail_to_serials.rb

```diff
@@ -0,0 +1,5 @@
+class AddThumbnailToSerials < ActiveRecord::Migration
+  def change
+    add_column :serials, :thumbnail, :string, null: false
```

Comment:
However serials are the feature not released yet, can we remove all record that thumbnail is `NULL`?